pywbem/_listener.py

```diff
@@ -1250,13 +1265,15 @@ def _create_destination(server, dest_url):
     dest_inst['CreationClassName'] = classname
     dest_inst['SystemCreationClassName'] = 'CIM_ComputerSystem'
     dest_inst['SystemName'] = getfqdn()
-    dest_inst['Name'] = 'cimlistener%d' % time.time()
+    dest_inst['Name'] = '%spywbemlistener%s' % (server.listener_id,
+                                                uuid.uuid4())
     dest_inst['Destination'] = dest_url
 
     dest_path = server.conn.CreateInstance(dest_inst)
     return dest_path
 
-def _create_filter(server, source_namespace, query, query_language):
+def _create_filter(server, source_namespace, query, query_language, \
+                   indication_id_prefix=None):
```

Comment:
The `indication_id_prefix` parameter is not used in `add_dynamic_filter()` which is the only place that calls this internal function. As a result, the user so far has no way to define it.

If it is really necessary to have this ID in addition to the listener ID, it should be externalized in `add_dynamic_filter()`. If not, it should be removed.